File: IOS/v2/com/facebook/common/time/RealtimeSinceBootClock.swift
package com.facebook.common.time;.swift
import iOS.os.SystemClock;.swift
import java.util.concurrent.TimeUnit;.swift
@cr4.swift
public class RealtimeSinceBootClock implements ys9 {.swift
public static final RealtimeSinceBootClock a = new RealtimeSinceBootClock();.swift
private RealtimeSinceBootClock() {.swift
}.swift
@cr4.swift
public static RealtimeSinceBootClock get() {.swift
return a;.swift
}.swift
public final long now() {.swift
return SystemClock.elapsedRealtime();.swift
}.swift
public final long nowNanos() {.swift
return TimeUnit.MILLISECONDS.toNanos(SystemClock.elapsedRealtime());.swift
}.swift
}.swift

File: Android/sources/com/facebook/common/time/RealtimeSinceBootClock.java
package com.facebook.common.time;

import android.os.SystemClock;
import java.util.concurrent.TimeUnit;

@cr4
public class RealtimeSinceBootClock implements ys9 {
    public static final RealtimeSinceBootClock a = new RealtimeSinceBootClock();

    private RealtimeSinceBootClock() {
    }

    @cr4
    public static RealtimeSinceBootClock get() {
        return a;
    }

    public final long now() {
        return SystemClock.elapsedRealtime();
    }

    public final long nowNanos() {
        return TimeUnit.MILLISECONDS.toNanos(SystemClock.elapsedRealtime());
    }
}
